store/sqlstore/channel_store.go

```diff
@@ -1038,35 +1038,32 @@ func (s SqlChannelStore) GetMoreChannels(teamId string, userId string, offset in
 	})
 }
 
-func (s SqlChannelStore) GetPublicChannelsForTeam(teamId string, offset int, limit int) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		data := &model.ChannelList{}
-		_, err := s.GetReplica().Select(data, `
-			SELECT
-			    Channels.*
-			FROM
-			    Channels
-			JOIN
-			    PublicChannels pc ON (pc.Id = Channels.Id)
-			WHERE
-			    pc.TeamId = :TeamId
-			AND pc.DeleteAt = 0
-			ORDER BY pc.DisplayName
-			LIMIT :Limit
-			OFFSET :Offset
+func (s SqlChannelStore) GetPublicChannelsForTeam(teamId string, offset int, limit int) (*model.ChannelList, *model.AppError) {
+	data := &model.ChannelList{}
```

Comment:
nit: Probably we can rename it to `channels` instead of `data`

Reply:
Renamed to `channels`